library/pkcs7.c

```diff
@@ -212,17 +175,12 @@ static int pkcs7_get_certificates( unsigned char **p, unsigned char *end,
      * The behaviour would be improved with addition of multiple signer support.
      */
     if ( end_cert != end_set )
-    {
```

Comment:
Since we have new code style standard: https://mbed-tls.readthedocs.io/en/latest/kb/how-to/rewrite-branch-for-coding-style/ , seems we have no need to remove the braces here.